lib/iris/unit.py

```diff
@@ -1823,9 +1823,9 @@ def convert(self, value, other, ctype=FLOAT64):
             >>> import numpy as np
             >>> c = unit.Unit('deg_c')
             >>> f = unit.Unit('deg_f')
-            >>> print(c.convert(0, f))
+            >>> print(np.round(c.convert(0, f), 8))
             32.0
-            >>> c.convert(0, f, unit.FLOAT32)
+            >>> np.round(c.convert(0, f, unit.FLOAT32), 8)
```

Comment:
I thought this one prints `32.0` on both Python 2 and 3? I only tested with `cf_units` on py3k though, but I don't expect it to differ from iris in this regard>